packages/sitecore-jss-react/src/components/Image.test.tsx

```diff
@@ -115,7 +115,7 @@ describe('<Image />', () => {
 
     it('should render <img /> with style and className props', () => {
       expect(img.getAttribute('style')).to.equal('width:100%');
-      expect(img.getAttribute('class')).to.equal(props.className);
+      expect(img.getAttribute('className')).to.equal(props.className);
```

Comment:
This _className_ attribute can't be provided as an output, since it's used only in JSX. In HTML we need to have _class_. You can double-check if the class is applied using _className_, but I think it shouldn't

Reply:
ahh it got converted back to _className_ through my function.
Updated my function now.